### DISCUSSION TITLE
plugin-form-builder: Add localized property to form title

### DISCUSSION THREAD

**Discussion by leikoilja** [payloadcms/payload#5592] [Feature Requests & Ideas]
A common use case would be to create a named form, for example 'Registration Form', which title must be localized to display the form title on the UI for the end user.

IMHO, we should add `localized: true` to the form's title field:
https://github.com/payloadcms/plugin-form-builder/blob/139aedab933edcd7b6f8b0179d2de81fe5606688/src/collections/Forms/index.ts#L79

**Comment by jacobsfletch** [comment]
This plugin is now being maintained in the [Packages Directory](https://github.com/payloadcms/payload/tree/main/packages/plugin-form-builder) of the [Payload Monorepo](https://github.com/payloadcms/payload). This repo will soon be archived and all open issues including this one will be closed. This issue, however, will be transferred over. Please refer to [this open discussion](https://github.com/payloadcms/payload/discussions/4466) for more details.

**Comment by DallasO** [comment]
Not only the title, but the Submit Button Label, Confirmation Message, and Custom Redirect URL are not able to be localized. A top level setting would be appreciated since adding the option to all available blocks/fields requires redefining all fields after a lot of work of finding the correct values.

**Comment by denolfe** [comment]
Moving this to Discussions as a feature request. We can continue the discussion there.
